float64[] joint_values
---
int64 error
